docs/elements/hx-beacon/index.html

```diff
@@ -0,0 +1,33 @@
+---
+title: <hx-beacon>
+minver: 0.22.0
+also:
+  components/beacon: Beacon
+---
+{% extends 'element.njk' %}
+{% block page_header %}
+  <p>
+    The custom <code>{{page.title}}</code> element provides an animation
+    that attracts a user's attention to a new feature
+    or advanced functionality in a control panel. A beacon disappears after a user clicks it.
+  </p>
+{% endblock %}
+
+{% block content %}
+  <section>
+    <dl class="metadata hxList">
+      <div>
+        <dt>Permitted Parents</dt>
+        <dd>Any that accept Phrasing Content</dd>
+      </div>
+      <div>
+        <dt>Permitted Children</dt>
+        <dd>Phrasing Content</dd>
+      </div>
+      <div>
+        <dt>Events</dt>
+        <dd><em>none</em></dd>
```

Comment:
Please add information about the `click` event.  Additionally, you will need to add a description in the `Methods` section.

Reply:
Changes have been done, please review